### ISSUE TITLE
Cover player improvements

### ISSUE THREAD

**Issue by digimezzo** [digimezzo/dopamine#679]
Reported by @yurisuika in #660.

I like the mini player, but I have a couple suggestions!

I think it would appear best if the title/artist, window controls, and gradients transitioned out of opacity when the window is not in focus. This would provide a much cleaner look for such a mode that is "focused" on the artwork.

As well, the song title is a bit large for such a small window. Even relatively short titles of several words are truncated at this scale. This is also an issue with Showcase mode to the same extent. I am sure it looks great for song titles that are a single word, but that is a very limited use case.

![image](https://github.com/user-attachments/assets/6c938114-9c61-4c5a-8baf-7eb9e805e85d)

**To summarize:**

- Fade out controls and title/artist and show the mon mouse move (like now playing)
- Decrease song title
- Improve fade for light covers
